keras/initializers.py

```diff
@@ -371,6 +371,28 @@ def he_normal(seed=None):
                            seed=seed)
 
 
+def selu_normal(seed=None):
```

Comment:
LeCun proposed this scheme way back (see ref in `lecun_uniform` docstring). Our initializers are named `*_normal` and `*_uniform`. We already have the `lecun_uniform` initializer, and this initializer is simply the normal version of it. I don't even know why didn't already have it, it's a significant inconsistency.